frontend/src/themes/transformThemeData.ts

```diff
@@ -0,0 +1,29 @@
+import defaultTheme from "./theme";
+
+export const transformThemeData = (data) => {
```

Comment:
transform theme data received from the API into a structured theme object

Reply:
Unsure about the name as  I missunderstood functions "task" when only reading the name :) But I do not have a better name. I'll just add some comment litterly describing it with your comment :D 
// transform theme data received from the API into a structured theme object

Additionally, maybe one could combine the `getHubTheme` with this function, as they are always called together